DEVELOPMENT.md

```diff
@@ -60,6 +60,17 @@ $ pytest --cov=cleanlab/ --cov-config .coveragerc --cov-report=html
 The coverage report will be available in `coverage_html_report/index.html`,
 which you can open with your web browser.
 
+
+### Type checking
+
+Cleanlab uses [mypy](https://mypy.readthedocs.io/en/stable/) typing. Type checking happens automatically during CI but can be tested locally.
+
+**Check typing in files:**
+
+```
+$ mypy <filename or foldername>
```

Comment:
Better to be specific here, that they should run this on the `cleanlab` directory. `$ mypy cleanlab`.

Reply:
Sounds great, wasn't sure if suggesting mypy can be used on individual files was necessary.

Reply:
In a larger project, perhaps it's useful, though for our project, type checking the entire thing takes like 3 seconds, so it's probably worth running the full type check every time.